KishanJ-Patel/One-Stop-Shopper
# Change spacing between buttons on the product card

### Context

- This pull request addresses issue OSS-35 which describes an uneven spacing bug between the plus button, minus button and the total quantity number on the the product card.
- This bug makes the user interface visually unappealing.

### Changes

- Changed the spacing between the plus button, the minus button and the total quantity number on the product card to space them evenly.

### Testing

- To test this bugfix:
  - Open the home page of the web app.
  - Add a product to the cart by clicking on the add-to-cart option if the cart is empty.
  - Observe the spacing between buttons on the product card.

### Related Issues

- Closes `OSS-35`

### Reviewer Requests

- @KishanJ-Patel : For frontend review
- @KishanJ-Patel : For backend review
- @KishanJ-Patel : For testing and QA
